DataFormats/Common/interface/WrapperBase.h

```diff
@@ -54,6 +55,14 @@ namespace edm {
 
     std::shared_ptr<soa::TableExaminerBase> tableExaminer() const { return tableExaminer_(); }
 
+    bool hasMemcpyTraits() const { return hasMemcpyTraits_(); }
+    bool hasMemcpyInit() const { return hasMemcpyInit_(); }
+    void memcpyInitialize(std::vector<size_t> const& parameters) { memcpyInitialize_(parameters); }
+    std::vector<size_t> memcpyParameters() const { return memcpyParameters_(); }
+    std::vector<std::pair<void const*, size_t>> memcpyRegions() const { return memcpyRegions_(); }
+    std::vector<std::pair<void*, size_t>> memcpyRegions() { return memcpyRegions_(); }
```

Comment:
I've used `std::vector<std::pair<void*, size_t>>` to support cases like a `PortableCollection::View`, where we may want to make copies column by column.

Other ides to improve the interface:
  - replace `void*` with `std::byte*` ?
  - replace `std::pair<void*, size_t>>` with `std::span<std::byte>` ?

Reply:
I've changed to `std::span<std::byte>`, which describes the intent better.